analyses/ssgsea-hallmark/01-run-ssgsea.R

```diff
@@ -0,0 +1,37 @@
+##########################################
+#Purpose: Code to run ssGSEA analysis
+#Author: Pichai Raman
+#Date: 9/23/2019
+##########################################
+
+
+#Call libraries
+library("GSVA")
+library("GSEABase")
+library("stringr")
+
+#Read in data 
+expData <- readRDS("../../data/pbta-gene-counts-rsem-expected_count.stranded.rds")
+
+#Format RNA-Seq data
+expData[,"max"] <- apply(expData[-1], FUN=max, MARGIN=1)
+expData[,"Gene"] <- stringr::word(as.character(expData[,1]), 2, sep = "_")
+expData <- expData[order(-expData[,"max"]),]
+expData <- expData[!duplicated(expData[,"Gene"]),]
+rownames(expData) <- expData[,"Gene"]
+expData <- expData[expData[,"max"]>0,]
+expData <- dplyr::select(expData, dplyr::starts_with("BS_"))
+expData <- expData[-1];
+
+#Log-Tranform for input into GSVA
+expData <- as.matrix(log2(expData+1))
+
+#Read Hallmark Gene Sets & Format
+hallmarkSets <- getGmt("references/h.all.v7.0.symbols.gmt", collectionType=BroadCollection(), geneIdType= SymbolIdentifier())
```

Comment:
Where are these functions coming from?

Reply:
This is from GSEABase, It allows you to read GMT files. I think you may be hinting at using the "::" function. I can update.